Makefile

```diff
@@ -0,0 +1,3 @@
+copy:
```

Comment:
Since we have a package.json, we will use a `npm` script for copying files rather than a Makefile.